### PR TITLE
Updating BoundedStaleness MaxStalenessIntervalInSeconds limits

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json [hunk 1]
```diff
@@ -625,10 +625,10 @@
           "format": "int64"
         },
         "maxIntervalInSeconds": {
-          "description": "When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
+          "description": "When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 5 - 86400. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
           "type": "integer",
-          "minimum": 1,
-          "maximum": 100,
+          "minimum": 5,
```
Comment (veronicagg) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json]: is this a bug fix? or could it mean a breaking change?

**Review by veronicagg — CHANGES_REQUESTED** [review]
Thanks @dmakwana , question inline.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json (hunk 1 above)
Comment (dmakwana) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json]: It's a bug fix

Comment (lmazuel) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json]: @veronicagg https://github.com/Azure/azure-cli/issues/4317

Comment (veronicagg) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json]: Cool, thanks for including more info!

**Review by veronicagg — APPROVED** [review]

**Pull request merged**